src/main/java/org/apache/commons/imaging/Imaging.java

```diff
@@ -197,6 +193,8 @@ public static boolean hasImageFileExtension(String filename) {
      * @return An ImageFormat, such as ImageFormat.IMAGE_FORMAT_JPEG. Returns
      *         ImageFormat.IMAGE_FORMAT_UNKNOWN if the image type cannot be
      *         determined.
+     * @throws org.apache.commons.imaging.ImageReadException
```

Comment:
Looks like an unrelated change to me. Please keep change sets as minimal as possible. It makes reviewing easier.